src/world/World.php

```diff
@@ -299,8 +300,8 @@ class World implements ChunkManager{
 	/** @phpstan-var \SplQueue<int> */
 	private \SplQueue $neighbourBlockUpdateQueue;
 	/**
-	 * @var true[] blockhash => dummy
-	 * @phpstan-var array<BlockPosHash, true>
+	 * @var int[] blockhash => dummy
```

Comment:
```suggestion
	 * @var int[] blockhash => update reference block flags
```

Reply:
where does "reference" come into it?

Reply:
Its mean the block referent for the update, no?

Reply:
That's a very confusing way of putting it. "cause flags" would be simpler.